### ISSUE TITLE
Socket.io update required to later versions as vulnerability reported

### ISSUE THREAD

**Issue by RMutharaju** [node-red/node-red-dashboard#697]
### What are the steps to reproduce?
With Socket.io: ~3.0.0 and earlier versions, there has a vulnerability reported for one of its dependency components
https://nvd.nist.gov/vuln/detail/CVE-2021-31597

### Please tell us about your environment:

- [ ] Node-RED-Dashboard version: Latest
- [ ] Node-RED version: 1.2.x
- [ ] node.js version:
- [ ] npm version:
- [ ] Platform/OS:
- [ ] Browser:

![image](https://user-images.githubusercontent.com/37263240/116189327-693bad00-a746-11eb-9e80-c58e9452a3cb.png)

**Comment by dceejay** [comment]
Yes - we are well aware of this. Currently the core of Node-RED and Dashboard both still support Nodev8 and 10 - moving to that new version will break support for them. We are currently in the process of moving to Node-RED v2 and at that point we will also release Dashboard v3 which will have tis fix.

**Comment by RMutharaju** [comment]
Thanks for you response @dceejay :)
